tests/Gaufrette/Functional/Adapter/AwsS3Test.php

```diff
@@ -81,73 +81,104 @@ private function createFilesystem(array $adapterOptions = [])
     }
 
     /**
+     * @test
      * @expectedException \RuntimeException
      */
-    public function testThrowsExceptionIfBucketMissingAndNotCreating()
+    public function shouldThrowExceptionIfBucketMissingAndNotCreating()
     {
         $this->createFilesystem();
         $this->filesystem->read('foo');
     }
 
-    public function testWritesObjects()
+    /**
+     * @test
+     */
+    public function shouldWriteObjects()
     {
         $this->assertEquals(7, $this->filesystem->write('foo', 'testing'));
     }
 
-    public function testChecksForObjectExistence()
+    /**
+     * @test
+     */
+    public function shouldCheckForObjectExistence()
     {
         $this->filesystem->write('foo', '');
         $this->assertTrue($this->filesystem->has('foo'));
     }
 
-    public function testGetsObjectUrls()
+    /**
+     * @test
+     */
+    public function shouldGetObjectUrls()
     {
         $this->assertNotEmpty($this->filesystem->getAdapter()->getUrl('foo'));
     }
 
-    public function testChecksForObjectExistenceWithDirectory()
+    /**
+     * @test
+     */
+    public function shouldCheckForObjectExistenceWithDirectory()
     {
         $this->createFilesystem(['directory' => 'bar', 'create' => true]);
         $this->filesystem->write('foo', '');
 
         $this->assertTrue($this->filesystem->has('foo'));
     }
 
-    public function testGetsObjectUrlsWithDirectory()
+    /**
+     * @test
+     */
+    public function shouldGetObjectUrlsWithDirectory()
     {
         $this->createFilesystem(['directory' => 'bar']);
         $this->assertNotEmpty($this->filesystem->getAdapter()->getUrl('foo'));
     }
 
-    public function testListKeysWithoutDirectory()
+    /**
+     * @test
+     */
+    public function shouldListKeysWithoutDirectory()
     {
         $this->assertEquals([], $this->filesystem->listKeys());
         $this->filesystem->write('test.txt', 'some content');
         $this->assertEquals(['test.txt'], $this->filesystem->listKeys());
     }
 
-    public function testListKeysWithDirectory()
+    /**
+     * @test
+     */
+    public function shouldListKeysWithDirectory()
     {
         $this->createFilesystem(['create' => true, 'directory' => 'root/']);
         $this->filesystem->write('test.txt', 'some content');
         $this->assertEquals(['test.txt'], $this->filesystem->listKeys());
         $this->assertTrue($this->filesystem->has('test.txt'));
     }
 
-    public function testKeysWithoutDirectory()
+    /**
+     * @test
+     */
+    public function shouldKeysWithoutDirectory()
     {
         $this->filesystem->write('test.txt', 'some content');
         $this->assertEquals(['test.txt'], $this->filesystem->keys());
     }
 
-    public function testKeysWithDirectory()
+    /**
+     * @test
+     */
+    public function shouldKeysWithDirectory()
```

Comment:
`shouldListKeysWithDirectory`

Reply:
This change will be broken because it's duplicated.

Reply:
same here ;)

Reply:
I think we can change into `shouldGetKeysWithDirectory` :).